src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetCohorts/WhenIValidateTheRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Queries.GetCohorts;
+using System;
+using System.Linq;
+
+namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetCohorts
+{
+    public class WhenIValidateTheRequest
+    {
+        private GetCohortsRequestValidator _validator;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _validator = new GetCohortsRequestValidator();
+        }
+
+        [Test]
+        public void ThenShouldReturnValidIfRequestIsValid()
+        {
+            //Act
+            var result = _validator.Validate(new GetCohortsRequest {AccountId = 123 });
+
+            //Assert
+            Assert.IsTrue(result.IsValid());
+        }
+
+        [Test]
+        public void ThenShouldReturnInValidIfRequestIsValid()
```

Comment:
Same question as above? Is the request valid or invalid?

Reply:
Renamed to ThenShouldReturnInValidIfAccountIdNotSet